CMakeLists.txt

```diff
@@ -981,6 +981,7 @@ option(USE_SSLS_EXPORT "Enable SSL session export support" OFF)
 if(USE_SSLS_EXPORT)
   if(_ssl_enabled)
     message(STATUS "SSL export enabled.")
+    set(USE_SSLS_EXPORT 1)
```

Comment:
`USE_SSLS_EXPORT` is set to 1 (= `ON`) at this point. This line can be safely deleted.